mscore/file.cpp

```diff
@@ -2356,11 +2375,13 @@ Score::FileError readScore(MasterScore* score, QString name, bool ignoreVersionE
                   { "gpx",  &importGTP                },
                   { "gp",   &importGTP                },
                   { "ptb",  &importGTP                },
+                  { "mscz,", 0                        },
```

Comment:
Why don't you follow the convention introduced here? You could define the method and pass its name here so that you don't need to write `if-else` at Line 2402

Reply:
This is because the functions probably need the `ignoreVersionError` parameter introduced below since it's basically opening score files... Am I wrong?

Reply:
I think it’s better to add these types not here, but higher, for example after
``` 
      if (suffix == "mscz" || suffix == "mscx") {
         ...
      else if (suffix == "mscz," || suffix == "mscx,") {
```

Reply:
I agree, and I tried doing so, but for some reasons the `setAutosaveDirty(false)` operation doesn't seem to take effect... Do you know what's going on?

Reply:
I don’t know, I tried to find out, but my debugger is crazy about such large files and method implementations in .h files, until I can find out

Reply:
Investigated behavior, `setAutosaveDirty (false)` is working correctly  
(if you open the file and do nothing else)

Reply:
But before I addressed your suggestion, it'd been working as expected (trying to save the file will pop out the Save As box), but after this change it no longer happens. I don't understand because basically I just moved some code here and there, no idea why it isn't working anymore...

Reply:
Aha! Turns out `setCreated(true)` is what was skipped.